### ISSUE TITLE
Missing syntax does not lead to warning

### ISSUE THREAD

**Issue by WJWH** [colah/ImplicitCAD#107]
The following extopenscad code works as expected:

```
difference(){
    cylinder(r=5,h=20);
    cylinder(r=2,h=20);
}
```

However, this code (with missing semicolon after the first cylinder) gives just the first cylinder and discards the second one without giving a warning.

```
difference(){
    cylinder(r=5,h=20)
    cylinder(r=2,h=20);
}
```

**Comment by julialongtin** [comment]
Still an issue. TL;DR: the second cylinder is seen as an argument to the first, not consumed, and no warning is generated over the unconsumed argument.

**Comment by julialongtin** [comment]
Error at line 2, column 3: Suite provided, but module "cylinder" does not accept one. Perhaps a missing semicolon?

**Comment by julialongtin** [comment]
Checking for this kind of error exists now. I'll close this issue when it lands on github, and i can provide a commit hash.

**Comment by julialongtin** [comment]
fixed, commit id b507c7a490cb65d5cc547d2bd2b4220820307619 . thanks!
